db/schema.rb

```diff
@@ -10,10 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-
-ActiveRecord::Schema.define(version: 2021_05_25_183405) do
-
-
+ActiveRecord::Schema.define(version: 2021_05_10_202141) do
```

Comment:
This needs to be the schema version that is currently in master. There was another change after yours `2021_05_25_183405`

Reply:
Oopsie copy/paste error, fixed ✔️